mods/ra/rules/civilian.yaml

```diff
@@ -244,8 +244,9 @@ V19.Husk:
 
 BARL:
 	Inherits: ^TechBuilding
-	Selectable:
-		Priority: 0
+	-Selectable:
+	CustomSelectionSize:
+		CustomBounds: 24,24
```

Comment:
The barrel's art isn't centered on the cell, so I felt this was better from a gameplay perspective.